sources/packages/web/src/services/http/Institution.ts

```diff
@@ -195,4 +197,42 @@ export class InstitutionApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async getPaginatedAESTInstitutionProgramsSummary(
+    institutionId: number,
+    pageSize: number,
+    page: number,
+    sortColumn: string,
+    sortOrder: SortDBOrder,
+    searchName: string,
+  ): Promise<AESTInstitutionProgramsSummaryPaginatedDto> {
+    const sortByOrder = sortOrder === SortDBOrder.ASC ? "ASC" : "DESC"; //Default sort order
+    try {
+      let queryString = "";
+      if (searchName) {
+        queryString += `searchProgramName=${searchName}&`;
+      }
+      if (sortColumn) {
+        queryString += `sortColumn=${sortColumn}&`;
+      }
+      if (sortByOrder) {
+        queryString += `sortOrder=${sortByOrder}&`;
+      }
+      if (pageSize) {
+        queryString += `pageSize=${pageSize}&`;
+      }
+      queryString += `page=${page}&`;
+      const response = await this.apiClient.get(
+        `institution/offering/institution/${institutionId}/programs?${queryString.slice(
+          0,
```

Comment:
i didn't get this part guru, why are you slicing?

Reply:
at the end of the query, there will be a & in the end, so slicing it at the end.